examples/cql-examples.json

```diff
@@ -63,11 +63,11 @@
   ],
   "Nested Property": [
     {
-      "name": "bin_maxbins",
+      "name": "bin-maxbins",
       "title": "Histogram - maxbins?"
     },
     {
-      "name": "scaleType",
+      "name": "scale-type",
```

Comment:
It would not work if you only rename it in this JSON but does not rename the actual file.